MPhys/Single_Emitter/Write_binary_file.ipynb

```diff
@@ -45,7 +45,7 @@
     "\n",
     "#load data from original G4 output (csv format)\n",
     "column_names = [\"time (ps)\", \"x\", \"y\", \"z\", \"px\", \"py\", \"pz\", \"E (MeV)\"]\n",
-    "photon_data_full = pd.read_csv('/hepgpu6-data1/kevans/Photons_1000000_filtered.csv', names = column_names)\n",
+    "photon_data_full = pd.read_csv('./Photons_1000000_filtered.csv', names = column_names)\n",
```

Comment:
This particular file which I got from Keith is too large (it's ~400MB) to add to GitHub directly so we would need to come up with another place to store it.  We could plausibly look into GitHub LFS but I am not sure whether the university / Research IT might have a better solution.

Again this file should probably be in the csv directory as well.

Reply:
Issue https://github.com/UoMResearchIT/mitsuba3-manchester/issues/11 created to track using GitHub LFS if needed.